modules/@apostrophecms/page/index.js

```diff
@@ -702,6 +702,7 @@ database.`);
                   editRole: 'admin',
                   publishRole: 'admin'
                 },
+                permissions: {},
```

Comment:
Our fake doc type manager for a missing type wasn't quite complete enough to meet new assumptions about what a manager contains.